internal/extractors/aggregator.go

```diff
@@ -53,7 +53,9 @@ func Aggregator(target chan<- common.DataRecord, source <-chan common.DataRecord
 		case innosat.SPCont:
 			// Report error missing start packet
 			if !multiPackStarted {
-				sourcePacket.Error = errors.New("got continuation packet without a start packet")
+				sourcePacket.Error = errors.New(
```

Comment:
The traceability of these is handled elsewhere, or?

Reply:
*all* errors get traceability added in the callbacks